python/build/Dockerfile-Python-Rosetta-Build

```diff
@@ -0,0 +1,20 @@
+FROM python:3.10-alpine
```

Comment:
As discussed it would be better to remove the /build folder in a subsequent release